products/sle/main.pm

```diff
@@ -359,6 +359,16 @@ if (get_var('ENABLE_ALL_SCC_MODULES') && !get_var('SCC_MODULES')) {
     }
 }
 
+# define aytests repo for support server (do not override defined value)
+if (get_var('SUPPORT_SERVER_ROLES') =~ /aytest/ && !get_var('AYTESTS_REPO')) {
+    if (sle_version_at_least('15')) {
+        set_var('AYTESTS_REPO', 'http://download.suse.de/ibs/Devel:/YaST:/Head/SUSE_SLE-15_GA/');
+    }
+    else {
+        set_var('AYTESTS_REPO', 'http://download.suse.de/ibs/Devel:/YaST:/SLE-12-SP3/SLE_12_SP3/');
```

Comment:
So all SLE version before 12-SP3 use the same repo?

Reply:
There only 2 repos for aytests and likely there won't be more, that's why I've decided not to put additional checks. It may get renamed for SP4, but that's not yet even planned...